Documentation/WEB2WEB.md

```diff
@@ -0,0 +1,126 @@
+# Web2Web Integration
+
+Starting from version 2.2.0 of Kovalee iOS SDK, Web2Web is supported. 
+
+In order to works correctly the app needs to supports deeplinks that follow the format below (the Kovalee team will configure it for you):
+
+```
+https://appname.go.link/web2web?adj_t=123456&adj_linkme=1&user_id={{user.id}}
+```
+
+
+## Supporting deeplinks to makes the user premium
+
+### For SwiftUI apps only (easy way)
+
+Just add the view modifier `checkWebUserPremium` from KovaleeSDK to your main view.
+
+```swift
+import KovaleeSDK
+
+@main
+struct MyApp: App {
+
+    var body: some Scene {
+        WindowGroup {
+            ... // your implementation
+        }
+        .checkWebUserPremium(
+            readPasteboard: true, // the default value is true
+            isUserPremium: { premiumStatus in
```

Comment:
Not clear what this method is. If it is a callback, I suggest calling it `onUserPremiumChange`